nemo_text_processing/inverse_text_normalization/en/taggers/decimal.py

```diff
@@ -33,11 +41,16 @@ def get_quantity(decimal: 'pynini.FstLike', cardinal_up_to_hundred: 'pynini.FstL
     Args: 
         decimal: decimal FST
         cardinal_up_to_hundred: cardinal FST
+        input_case: accepting either "lower_cased" or "cased" input.
     """
     numbers = cardinal_up_to_hundred @ (
         pynutil.delete(pynini.closure("0")) + pynini.difference(NEMO_DIGIT, "0") + pynini.closure(NEMO_DIGIT)
     )
-    suffix = pynini.union("million", "billion", "trillion", "quadrillion", "quintillion", "sextillion")
+    suffix = pynini.union("million", "billion", "trillion", "quadrillion", "quintillion", "sextillion",)
+
+    if input_case == INPUT_CASED:
+        suffix |= pynini.union("Million", "Billion", "Trillion", "Quadrillion", "Quintillion", "Sextillion",)
```

Comment:
can we put this in a tmp variable? so we don't define it twice?

Reply:
found a .tsv with these values